_posts/2022-09-09-snapshot-operations.md

```diff
@@ -0,0 +1,194 @@
+---
+layout: post
+title:  "Snapshot Operations in OpenSearch"
+authors:
+  - adnapibar
+  - kotwanikunal
+  - nbower
+date:   2022-09-09 00:00:00 -0700
+categories:
+  - technical-post
+---
+
+# Snapshot operations in OpenSearch
```

Comment:
One small edit - can you remove this extra title/header on line 13? It ends up looking doubled when published since there's already the same text in the "title" field above.

Reply:
Thanks for pointing that out. Updated the PR.